packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -58,21 +58,27 @@ export const itemToSummaryItem = (item: Item): InboundShipmentItem => {
   };
 };
 
+export enum ModalMode {
+  Create,
+  Update,
+}
+
```

Comment:
I haven't made this change in the detail view for outbounds, because i'm not sure that the behaviour of disabling the item input when youre in update mode is wanted?